Backend/NoCode.FlowerShop.Api/Controllers/FlowersController.cs

```diff
@@ -15,4 +19,13 @@ public FlowersController(ISender mediator, IMapper mapper)
         _mediator = mediator;
         _mapper = mapper;
     }
+
+    [HttpPost("")]
+    [AuthorizeRoles(UserRole.Administrator)]
+    public async Task<IActionResult> Create(CreateFlowerRequest request)
+    {
+        var query = _mapper.Map<CreateFlowerCommand>(request);
+        var result = await _mediator.Send(query);
+        return result.Match(Ok, Problem);
```

Comment:
You should return a separate response (defined in Contracts layer), not the result from Application layer.